posts/month-in-mathlib-aug-2021.md

```diff
@@ -0,0 +1,74 @@
+---
+author: 'Mathlib community'
+category: 'month-in-mathlib'
+date: 2021-08-31 08:56:51+00:00
+description: ''
+has_math: true
+link: ''
+slug: month-in-mathlib-aug-2021
+tags: ''
+title: This month in mathlib (Aug 2021)
+type: text
+---
+
+# This month in mathlib (Aug 2021)
+
+## Highlighted PRs
+
+* [PR #8652](https://github.com/leanprover-community/mathlib/pull/8652) :: chore(*): update lean to 3.32.1
+
+  The community fork of Lean made two new
+  [releases](https://github.com/leanprover-community/lean/blob/master/doc/changes.md#3321c-12-august-2021)
+  `3.32.0` and `3.32.1`.
+  This is part of the preparations for porting mathlib to Lean 4.
+
+* [PR #8281](https://github.com/leanprover-community/mathlib/pull/8281) :: feat(topology,geometry/manifold): continuous and smooth partition of unity
+
+  See the [companion blogpost](continuous-partitions-of-unity/) for details.
+
+* Radon-Nikodym and Lebesgue decomposition. The four PRs
+  [PR #8645](https://github.com/leanprover-community/mathlib/pull/8645)
+  [PR #8687](https://github.com/leanprover-community/mathlib/pull/8687)
+  [PR #8763](https://github.com/leanprover-community/mathlib/pull/8763)
+  [PR #8875](https://github.com/leanprover-community/mathlib/pull/8875)
+  together contribute
+  the Lebesgue decomposition for sigma-finite measures
+  and the Radon-Nikodym theorem.
+
+* [PR #7978](https://github.com/leanprover-community/mathlib/pull/7978) :: feat(measure_theory/interval_integral): strong version of FTC-2
+
+  This PR signals the start of a serious attack on Stokes' theorem,
```

Comment:
Are you sure this is related to Yury's work on Stokes?

Reply:
No I'm not sure. I strongly got that impression, but now that you ask, I'm not sure. @urkud could you confirm?

Reply:
I don't think Yury uses this (he gives a direct proof using Henstock integrals). So maybe don't mention Stokes, but just say that this weakens considerably the assumptions of FTC-2, replacing continuity of the derivative by the much more natural assumption of integrability.